Cargo.toml

```diff
@@ -38,18 +38,29 @@ rayon = "1.5.1"
 ff = "0.11"
 group = "0.11"
 pasta_curves = "0.3"
-rand = "0.8"
+rand_core = { version = "0.6", default-features = false }
 blake2b_simd = "1"
 
 # Developer tooling dependencies
 plotters = { version = "0.3.0", optional = true }
 tabbycat = { version = "0.1", features = ["attributes"], optional = true }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+# plotters depends on web-sys, which eventually depends on bumpalo 3. This dependency is
+# required because our MSRV is 1.51, but bumpalo 3.9 increased its MSRV to 1.54. We can
+# remove this once our MSRV is 1.54+ (and should do so, because currently this makes it a
+# required dependency even if the dev-graph feature flag is not enabled).
+bumpalo = "=3.8.0"
```

Comment:
This seems too strict, we should allow for e.g. `3.8.1`. I forgot how to do that with semver.

Reply:
Looks like that's done with `~3.8.0`.

Reply:
[This is now](https://github.com/zcash/halo2/compare/f68dfe4a88102cfb774a49c636725187e75dd3f6..9e63eeff1b2c4407bd9dfd714d5a0a44b275cbc9) `bumpalo = ">=3,<3.9.0"`.